las/credentials.py

```diff
@@ -1,68 +1,108 @@
 import configparser
-
 import os
-from os.path import expanduser, exists
-from typing import Tuple, Optional
+import time
+from os.path import exists, expanduser
+from typing import Optional, Tuple, List
+
+import requests
+from requests.auth import HTTPBasicAuth
 
 
 class MissingCredentials(Exception):
     pass
 
 
 class Credentials:
-    """Used to fetch and store credentials. One of 4 conditions must be met to successfully create credentials.
+    """Used to fetch and store credentials. One of 3 conditions must be met to successfully create credentials.
 
     1. credentials_path is provided
-    2. access_key_id, secret_access_key and api_key is provided
-    3. credentials is located in default path ~/.lucidtech/credentials.cfg
-    4. the following environment variables are present:
-        - LAS_ACCESS_KEY_ID
-        - LAS_SECRET_ACCESS_KEY
```

Comment:
Can we explain in which order it will look for credentials? I suggest: 1) If they are provided directly to the class 2) path to credentials are provided directly 3) ENV 4) credentials file at default location